recipes/blockbuster/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+make
+mkdir $PREFIX/bin
+cp $SRC_DIR/blockbuster.x $PREFIX/bin
```

Comment:
@johanneskoester it is really wired - it actually works locally.
The difference seems to be that if you install into $PREFIX it ends up in `CONDA/bin` instead of `CONDA/envs/env_name/bin`
@dmaticzka sorry, we need `bin` :(